# 03-Bacterial-Coinfections

To better understand the immune response induced by Influenza viruses and bacterial pathogens, we developed in vitro models to analyze the changes in transcriptomic (RNA-Seq) and proteomic (OLINK) profiles of bone marrowderived macrophages (BMDMs) in this context. In this project, we described the results of our Coinfection and Superinfection models with the most common human pathogen Streptococcus pneumoniae (Spn). Here, macrophages were differentiated with GM-CSF for 7 days and then coinfected (simultaneously for 8h) at a multiplicity of infection (MOI) of 1 with Influenza A/PR/8/34 (PR8) and Spn (ATCCÂ®6301) or superinfected (48h of PR8 followed by 8h with Spn). The results showed that in coinfection, the immune response is characterized by the synergistic expression of cytokines and chemokines among other genes, mainly driven by the bacterial infection. In particular, NF-ÎºB-dependent genes were significantly correlated with this synergistic effect.  However, in the case of Superinfection, we observed that the response was mainly driven by the preceding viral infection, but exacerbated by bacterial Superinfection. In particular, genes related to fibrin clot formation and amplification of coagulation pathways were exacerbated, 
potentially contributing to the increased severity of the consequences of Superinfection. 

Bacterial coinfection was explored using *Streptococcus suis*, *Streptococcus porci^*, and other *Streptococcus pneumoniae* serotypes, highlighting how BMDMs adaptively modulate responses to distinct pathogens in secondary infections. Additionally, BMDM responses to both human and swine-adapted influenza subtypes varied by HA subtype in coinfection scenarios. Age effects were also examined in BMDMs differentiated with M-CSF across newborn, adult, and aged mice. Findings showed age-dependent gene expression patterns with proinflammatory cytokines being predominant in neonates during coinfection, while older mice exhibited higher chemokine expression and inflammation during Superinfection. This in vitro model offers valuable insights into age- and subtype-specific inflammatory responses to influenza-bacterial coinfections. 
 
On the other hand, despite the known inflammatory response observed in hospitalized patients, it has been proved that vaccination against circulating Influenza strains reduce those hospitalizations and increase survival rates. Thus, we further tested the efficacy of TIV (trivalent influenza vaccine) vaccination on mortality, morbidity, and pathogen loads in the context of secondary bacterial infections. For this purpose, we developed a Co/Superinfection mouse model using A/New Caledonia/20/1999 (H1N1) Influenza virus and a highly invasive Streptococcus pneumoniae (serotype 1) strain. Mice received either a sub-lethal dose of virus, bacteria, or both, with bacteria administered on the same day as the virus (coinfection) or 7 days later (Superinfection). TIV vaccine was administered 21 days before infection. Vaccination significantly reduced death rates across both co- and Superinfection groups. Immune responses were analyzed through pulmonary viral/bacterial titers, immune cell infiltration (flow cytometry), cytokine production (Luminex), and IgG subtype proportions, indicating that both coinfection and Superinfection elevated mortality compared to single infections. Our model demonstrates that TIV mitigates the severity of these complex infection dynamics, highlighting the importance of vaccination in reducing influenza-associated bacterial complications, especially in high-risk populations. 

Taken together, these findings underscore the broad impact of influenza on both microbiota dynamics and immunity and highlight the critical role of vaccination in mitigating the compounded risks of secondary bacterial infections and variant virus strains in vulnerable populations. 

## ðŸŽ¯ Objectives

- To develop and characterize an in vitro viral-bacterial coinfection model in bone marrow-derived macrophages to investigate immune responses during Influenza and Spn Coinfection, with transcriptomic and 
proteomic profiling to understand inflammatory dynamics.
- To test the vaccine's impact in experimental in vivo murine Coinfection and Superinfection models to assess the vaccineâ€™s capacity to mitigate mortality and immune complications from secondary bacterial infections. 


 ## ðŸ”¬ Metodology

Bone marrow-derived macrophages (BMDMs) were isolated from wild-type C57BL/6J mice aged 6â€“8 weeks and used for *In vitro* experiments. For cell differentiation, GM-CSF cytokine (GibcoTM, Waltham, MA, EEUU) (25ng/mL) was added to the media. For both Coinfection and Superinfection experiments, a Multiplicity of Infection (MOI) of 1 was used. in the case of the bacterial,  Serotype 1 Streptococcus pneumoniae ATCCÂ® 6301 was used. Bacteria were grown and maintained at 37Â°C, 5% CO2 on Blood agar plates.

- COINFECTION
Differentiated BMDMs were washed 3 times with PBS to eliminate any possible FBS remains. Fresh RPMI media without antibiotics were added, including the corresponding mock or MOI 1 viral load. For bacterial infection, as the volume containing the bacteria was lower than 20 ÂµL, saline solution or previously mixed bacteria were directly added to each corresponding wells. Cells were incubated for 8h prior to supernatant collection and cell lysis using TRIzol Â® (Thermo Scientific, Waltham, MA, USA).

- SUPERINFECTION
Similar to Coinfection, BMDMs were washed 3 times. On day 7, fresh RPMI media without antibiotics were added to mock and bacterial conditions. For Influenza infection, fresh RPMI containing Influenza virus was added to the wells for 24 or 48 hours. Bacteria from fresh cultures were added directly to the corresponding bacterial and superinfected wells for 8h. After incubation at 37ÂºC, supernatant was collected, and macrophages were lysed using TRIzol Â® (Thermo Scientific, Waltham, MA, USA).

RNA-Sequencing was conducted with the Illumina NextSeq 500/550 High Output Kit v2.5 (single-end, 75 cycles). Samples were randomized and processed across four sequencing runs. The Differential analysis was performed with DeSeq2 R package under standard parameters. Genes were considered significantly expressed if they showed an adjusted p-value (Bonferroni-Hochberg multiple testing, padj) lower than 0.05 and a Fold Changeâ€‰>â€‰2.  
Supernatants were also collected during the experimental procedures at 8h in Coinfection, and 48h or 48h+8h in Superinfection were stored at 80ÂºC. Then, the samples were processed following the manufacturerâ€™s instructions for OlinkÂ® Target 96 Inflammation Cytokine panel. 

## Main conclusions

TThe work presented here presents the value of exploring *In vitro* models to examine some characteristics of the immune response that can help in understanding complex scenarios, in this case, the comparison between Coinfection and Superinfection in macrophages differentiated with GM-CSF and M-CSF. The observed synergistic gene expression patterns (particularly those linked to NF-ÎºB-dependent inflammation in Coinfections and coagulation associated patterns in Superinfections) reveal important insights into the mechanisms that may contribute to severe clinical outcomes, such as cardiac failure or cardiovascular-related problems.

> **Contacto:** Javier Arranz Herrero | [LinkedIn](https://www.linkedin.com) | [Email](mailto:j.arranzherrero@gmail.com)
